src/redux/reducers/index.js

```diff
@@ -104,16 +99,11 @@ export const dateRangeFilterReducer = (state = preloadSelectedTimelineRange, act
   switch (action.type) {
     case actionTypes.CREATE_COLLECTION:
     case actionTypes.SELECT_COLLECTION:
+    case actionTypes.DELETE_COLLECTION:
+    case actionTypes.CLEAR_COLLECTION:
```

Comment:
this is cleanup from the pivot to consolidating `CollectionPreview` with `CatalogScreen`